activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ModelExtensionsService.java

```diff
@@ -37,22 +37,28 @@
     public ModelExtensionsService(Set<ModelExtensionsValidator> metadataValidators,
                                   ExtensionsModelValidator extensionsModelValidator,
                                   ModelTypeService modelTypeService) {
-        this.modelExtensionsValidatorsMapByModelType = metadataValidators.stream().filter(validator -> validator.getHandledModelType() != null)
-                .collect(Collectors.groupingBy(validator -> validator.getHandledModelType().getName()));
-
-        // Add the generic JSON extensions schema to all the avaliable model types
-        modelTypeService.getAvailableModelTypes().stream().forEach(modelType -> this.modelExtensionsValidatorsMapByModelType.put(modelType.getName(),
-                                                                                                                                 this.modelExtensionsValidatorsMapByModelType
-                                                                                                                                         .containsKey(modelType.getName())
-                                                                                                                                                 ? Stream.concat(this.modelExtensionsValidatorsMapByModelType
-                                                                                                                                                         .get(modelType.getName())
-                                                                                                                                                         .stream(),
-                                                                                                                                                                 Stream.of(extensionsModelValidator))
-                                                                                                                                                         .collect(Collectors
-                                                                                                                                                                 .toList())
-                                                                                                                                                 : Collections
-                                                                                                                                                         .singletonList(extensionsModelValidator)));
+    this.modelExtensionsValidatorsMapByModelType =
+        metadataValidators.stream()
+            .filter(validator -> validator.getHandledModelType() != null)
+            .collect(Collectors.groupingBy(validator -> validator.getHandledModelType().getName()));
 
+    // Add the generic JSON extensions schema to all the available model types except PROCESS
```

Comment:
why are you excluding the processes?

I think this file should remain unchanged.

Reply:
Actually there no relation to the model extensions schema (https://github.com/Activiti/activiti-cloud-modeling-service/blob/develop/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json). both are independent. it is affecting the change unnecessarily. so it removed

Reply:
You are right, it has changed a lot since the last time I worked with process extensions.